src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1865,4 +1889,20 @@
     <value>Non-monospace fonts:</value>
     <comment>This is a label that is followed by a list of proportional fonts.</comment>
   </data>
+  <data name="Profile_RainbowSuggestions.Header" xml:space="preserve">
+    <value>Display suggestions UI preview text in rainbow colors</value>
+    <comment>This is a label for a setting that, when enabled, applies a rainbow coloring to the preview text from the suggestions UI.</comment>
+  </data>
+  <data name="Globals_EnableColorSelection.Header" xml:space="preserve">
+    <value>Experimental: Add key bindings to color selected text</value>

```

Comment:
```suggestion
    <value>Experimental: Add key bindings to highlight selected text and to search for and highlight all instances selected text</value>
```

wording could use work, but more accurate

Reply:
Changed to "Experimental: Add key bindings to highlight selected text and to search for and highlight all instances of selected text" (there was a missing "of")

I think that works. We can iterate on it if we get feedback.

Reply:
Urgh, we have a problem:
![{E943A5D4-6A79-4161-BB3A-279F41755537}](https://github.com/user-attachments/assets/d62c3e0a-d938-400e-8ba9-eef2cabde590)

It's because of the SettingContainer styling. See inline comments below
```XML
<Style TargetType="local:SettingContainer">
    <Setter Property="Margin" Value="0,4,0,0" />
    <Setter Property="IsTabStop" Value="False" />
    <Setter Property="MaxWidth" Value="1000" />
    <Setter Property="Template">
        <Setter.Value>
            <ControlTemplate TargetType="local:SettingContainer">
                <Grid AutomationProperties.Name="{TemplateBinding Header}"
                      Style="{StaticResource NonExpanderGrid}">
                    <Grid.ColumnDefinitions>
                        <ColumnDefinition Width="*" />
                        <ColumnDefinition Width="Auto" />
                    </Grid.ColumnDefinitions>
                    <!--  This StackPanel is the problem! I'm guessing the StackPanel makes it so that wrapping property
                        isn't applied because we _technically_ never wrap (even though wrap is set on the text block). This doesn't occur with the HelpTextBlock below. Surprised this issue never came up for localization!  -->
                    <StackPanel Style="{StaticResource StackPanelInExpanderStyle}">
                        <StackPanel Orientation="Horizontal">
                            <TextBlock Style="{StaticResource SettingsPageItemHeaderStyle}"
                                       Text="{TemplateBinding Header}" />
                            <Button x:Name="ResetButton"
                                    Style="{StaticResource SettingContainerResetButtonStyle}">
                                <FontIcon Glyph="&#xE845;"
                                          Style="{StaticResource SettingContainerFontIconStyle}" />
                            </Button>
                        </StackPanel>
                        <TextBlock x:Name="HelpTextBlock"
                                   Style="{StaticResource SettingsPageItemDescriptionStyle}"
                                   Text="{TemplateBinding HelpText}" />
                    </StackPanel>
                    <ContentPresenter Grid.Column="1"
                                      HorizontalAlignment="Right"
                                      VerticalAlignment="Center"
                                      Content="{TemplateBinding Content}" />
                </Grid>
            </ControlTemplate>
        </Setter.Value>
    </Setter>
</Style>
```

Reply:
let's put experimental on the next line. or since it's a subtitle, `Experimental Setting`

Reply:
(or explain the whole setting in there. `Experimental: conhost-like Color Selection` `Adds blah blah blah to blah blah and also blah blah`

Reply:
Alright, I've reworded such that the header is shorter and I added additional details as the help text. Here's the result:
![{38337514-0674-4A00-BB59-07CE121CD142}](https://github.com/user-attachments/assets/8f513b44-e16f-4a65-ae97-1f26b434f4ec)